### PR TITLE
Switch to gradle, move to com.mirego.j2js

### PR DESCRIPTION
**Pull request by wjobin**

### CODE REVIEW

File: generator/build.gradle [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+dependencies {
+    compile project(':shared')
+    compile files("${System.properties['java.home']}/../lib/tools.jar")
+    compile 'com.google.guava:guava:18.0'
+    compile 'com.google.javascript:closure-compiler:v20141215'
+    compile 'org.mozilla:rhino:1.7R4'
+    compile 'com.google.code.findbugs:annotations:2.0.0'
+
+    compile 'junit:junit:4.8.1'
+    //testCompile project(':server')
```
Comment (Boubalou) [inline generator/build.gradle:25]: Needed?

**Comment by mbaron** [comment]
:+1: :+1: :tada:

File: generator/build.gradle (hunk 1 above)
Comment (wjobin) [inline generator/build.gradle:25]: I don't know... It was in pom.xml, I hadn't migrated server yet, comment to test and it worked.

**Comment by Boubalou** [comment]
Insanity!

**Pull request merged**